daq/http_server.py

```diff
@@ -63,9 +68,24 @@ def map_request(self, path, target):
     def get_data(self, path, opts):
         """Get data for a particular path"""
         try:
-            if path in self._paths:
-                return self._paths[path](opts)
+            for a_path in self._paths:
```

Comment:
What does path here refer to?

Reply:
request_path -- updated variable name and comment to clarify